test/integration/JsonSerialization.test.ts

```diff
@@ -10,6 +10,7 @@
   PrimitiveUri,
   ResourceChild,
 } from "../fixture/generatedTypeScript/fhir";
+import { ValueSet } from "../../generated/typescript/classes/ValueSet";
```

Comment:
Codacy found an issue: [Missing file extension for "../../generated/typescript/classes/ValueSet"](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=6104340)